src/lib/plugins/system-info.js

```diff
@@ -0,0 +1,517 @@
+// This library contains several chunks of code from Sebastian Hildebrandt's
+// systeminformation library, MIT
+// https://github.com/sebhildebrandt/systeminformation
```

Comment:
Hmm i'll have a look into that. Since we're compiling it with modules included and using only es6 import we should get only the functions we used in the compiled package. Otherwise I would not include it indeed :wink: